src/apps/patron-page/PatronPage.dev.tsx

```diff
@@ -70,12 +70,8 @@ export default {
       defaultValue: "Close pause reservations modal",
       control: { type: "text" }
     },
-    dateInputsStartDateLabelText: {
-      defaultValue: "Start date",
-      control: { type: "text" }
-    },
-    dateInputsEndDateLabelText: {
-      defaultValue: "End date",
+    pauseReservationModalDateRangeLabelText: {
```

Comment:
Is this a newly introduced text? If so, then we need a PR in dpl-cms as well with it 😉

Reply:
It's defined here:
https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/696